source/problem/SWE/kernels_preprocessor/swe_pre_init_data.hpp

```diff
@@ -285,8 +285,8 @@ void Problem::initialize_data_kernel(ProblemMeshType& mesh,
             uint element_1 = bound;
             uint element_2 = (bound + 1) % elt.data.get_nbound();
 
-            if ((elt.GetBoundaryType()[element_1] != SWE::BoundaryTypes::distributed) &&
```

Comment:
@kazbekkm is this correct? `IsDistributed` will also return true for distributed levees.